patches/rrweb@2.0.0-alpha.13.patch

```diff
@@ -1,812 +0,0 @@
-+                    atrule.params = atrule.params.replace(MEDIA_SELECTOR_GLOBAL, '$1-$2')
-+                }
-+            },
-+        }
-+    },
-+}
-+
-+// Adapted from https://github.com/giuseppeg/postcss-pseudo-classes/blob/master/index.js
-+const pseudoClassPlugin = {
-+    postcssPlugin: 'postcss-hover-classes',
-+    prepare: function () {
-+        const fixed = []
-+        return {
-+            Rule: function (rule) {
-+                if (fixed.indexOf(rule) !== -1) {
-+                    return
-+                }
-+                fixed.push(rule)
-+
-+                rule.selectors.forEach(function (selector) {
-+                    if (!selector.includes(':')) {
-+                        return
-+                    }
-+
-+                    const selectorParts = selector.replace(/\n/g, ' ').split(' ')
-+                    const pseudoedSelectorParts = []
-+
-+                    selectorParts.forEach(function (selectorPart) {
-+                        const pseudos = selectorPart.match(/::?([^:]+)/g)
-+
-+                        if (!pseudos) {
-+                            pseudoedSelectorParts.push(selectorPart)
-+                            return
-+                        }
-+
-+                        const baseSelector = selectorPart.substr(0, selectorPart.length - pseudos.join('').length)
-+
-+                        const classPseudos = pseudos.map(function (pseudo) {
-+                            const pseudoToCheck = pseudo.replace(/\(.*/g, '')
-+                            if (pseudoToCheck !== ':hover') {
-+                                return pseudo
-+                            }
-+
-+                            // Ignore pseudo-elements!
-+                            if (pseudo.match(/^::/)) {
-+                                return pseudo
-+                            }
-+
-+                            // Kill the colon
-+                            pseudo = pseudo.substr(1)
-+
-+                            // Replace left and right parens
-+                            pseudo = pseudo.replace(/\(/g, '\\(')
-+                            pseudo = pseudo.replace(/\)/g, '\\)')
-+
-+                            return '.' + '\\:' + pseudo
-+                        })
-+
-+                        pseudoedSelectorParts.push(baseSelector + classPseudos.join(''))
-+                    })
-+
-+                    const newSelector = pseudoedSelectorParts.join(' ')
-+                    if (newSelector && newSelector !== selector) {
-+                        rule.selector += ',\n' + newSelector
-+                    }
-+                })
-+            },
-+        }
-+    },
-+}
-diff --git a/es/rrweb/packages/rrweb-snapshot/es/rrweb-snapshot.js b/es/rrweb/packages/rrweb-snapshot/es/rrweb-snapshot.js
-index 38a23aaae8d683fa584329eced277dd8de55d1ff..8aeee467a3bab9baeefb1a97f2b131bedbd0fa3c 100644
---- a/es/rrweb/packages/rrweb-snapshot/es/rrweb-snapshot.js
```

Comment:
Now that https://github.com/rrweb-io/rrweb/pull/1458 we don't need any of the custom CSS stuff in a separate patch